src/components/product.js

```diff
@@ -233,11 +226,18 @@ export default class Product extends Component {
   }
 
   get variantExists() {
-    return Boolean(this.model.selectedVariant);
+    return Boolean(this.model.variants.find((variant) => {
```

Comment:
I think you could skip the explicit  `Boolean` and just swap `.find` for `.some`